src/lib/di.container.ts

```
@@ -0,0 +1,59 @@
+    this._container.bind(DITypes.AddonService).to(AddonService);
+    this._container.bind(DITypes.PlanService).to(PlanService);
+    this._container.bind(DITypes.SubscriptionService).to(SubscriptionService);
+    this._container.bind(DITypes.FeatureService).to(FeatureService);
+    this._container.bind(DITypes.OrganizationService).to(OrganizationService);
+    this._container.bind(DITypes.UserService).to(UserService);
+  }
+
+  public static getInstance<T extends keyof ServiceTypes>(type: T) {
+    return this._container.get<ServiceTypes[T]>(type) as ServiceTypes[T];
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)